### PR TITLE
Typo and naming fixes in the documentation

### PR DESCRIPTION
**Pull request by Vtec234**
A PR where I fix simple typos and naming errors that I find or which people report in issues.

### CODE REVIEW

**Comment by shy0013** [comment]
AFFIXED

File: doc/specs/vulkan/man/VkMemoryPropertyFlags.txt [hunk 1]
```diff
@@ -22,14 +22,14 @@ ename:VK_MEMORY_PROPERTY_HOST_VISIBLE_BIT::
     Identifies a memory pool that can be mapped into host memory address
     space and thus is accessible by the host.
 
-ename:VK_MEMORY_PROPERTY_HOST_NON_COHERENT_BIT::
+ename:VK_MEMORY_PROPERTY_HOST_COHERENT_BIT::
```
Comment (oddhack) [inline doc/specs/vulkan/man/VkMemoryPropertyFlags.txt]: Apparently we flipped the sense of those bits at some point and then didn't reflect it in the spec. Good catch, and it's obvious how to fix the text to match the name change, but this is an actual spec change so will have to go back to the working group.

**Comment by oddhack** [comment]
We're handling these internally and will update the public repo, hopefully next week. If you can, please split changes like these into smaller pull requests. I have authority to accept small typo changes on Github directly without cycling back through the Vulkan WG, but they need to be small and localized, and there's too much stuff going on at once here to bypass the process.

**Comment by Vtec234** [comment]
Done, split this PR into three: #23, #24, #25. Won't PR the enum corrections in VkMemoryPropertyFlagBits, since you're saying that this requires the intervention of the WG.

**Pull request closed without merging**